Gaming The Syllabus/Assets/Scripts/Riddles/UI/AnswerSubmissionManager.cs

```diff
@@ -8,11 +8,16 @@
 /// </summary>
 public class AnswerSubmissionManager : Singleton<AnswerSubmissionManager>
 {
+
+    public static bool isSubmitted;
+    public static bool isCorrect;
```

Comment:
Do these need to be public? I know we're setting isCorrect inside of Awake in SyllabusRiddleManager but I'm unsure why